src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -226,7 +238,28 @@ def _search(
                 query_kwargs.update(extra_query_kwargs)
 
             if query_kwargs["sort_by"] == "betterPriority":
-                query_kwargs["aggregate_kwargs"] = self.build_better_priority_sort_kwargs(request)
+                if features.has(
+                    "organizations:better-priority-sort-experiment",
+                    organization,
+                    actor=request.user,
+                ):
+                    choice = expt_manager.get(
+                        "PrioritySortExperiment", org=organization, actor=request.user
+                    )
+                    # force into variant1 for internal testing
+                    if features.has(
```

Comment:
@ceorourke You shouldn't need the experiment AND the feature flag, we should be ORing the feature flags, right?